hyperspy/tests/utils/test_roi.py

```diff
@@ -308,10 +308,39 @@ def test_out(self):
         d = s.data.sum()
         sr.data += 2
         nt.assert_equal(d + sr.data.size * 2, s.data.sum())
+        r.left += 2
+        r.right += 2
```

Comment:
Some future PR should definitely add `width` and `height` convenience properties to the appropriate ROIs...